tests/conftest.py

```diff
@@ -202,6 +202,12 @@ def satellite():
     return None
 
 
+@pytest.fixture(scope="class")
+def class_satellite_sca_disable(satellite):
```

Comment:
subscription cases need sca disable, but the default is enable.
will maintain the pipeline job to initial setup sca as requirement